pipelines/tekton/aiedge-e2e/tasks/retrieve-build-image-info.task.yaml

```diff
@@ -15,18 +15,24 @@ spec:
     type: string
   - name: pipeline-run-uid
     type: string
-  - name: target-registry-url
-    type: string
+  - name: target-image-tag-references
+    type: array
+  workspaces:
+  - name: images-url
   steps:
   - name: get-image-sha
     image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+    args:
+      - "$(params.target-image-tag-references[*])"
```

Comment:
Can't reference array param directly in script, so need to convert to `args`, then use `$@` in script.